tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -36,6 +36,72 @@
     ASSERT_TRUE([backingElement isKindOfClass:[WXFrameworkElement class]]);
 }
 
+@interface UIDeallocTestButton : UIButton {
+    NSCondition* _condition;
+    BOOL* _signaled;
```

Comment:
? so we can set the correct variable to signaled. 

---
In reply to: [98299381](https://github.com/Microsoft/WinObjC/pull/1861#discussion_r98299381) [](ancestors = 98299381)